workshop/Lab4/osbapi.md

```diff
@@ -0,0 +1,68 @@
+# Open Service Broker API
```

Comment:
I feel like moving all this info to an aside is going to mean nobody reads it.

Reply:
Was Doug's idea. I don't think knowing what's going on behind the scenes is necessary to get up and running.

Reply:
Perhaps we should include just a small number of paragraphs covering the concepts (key kube resources) in the main doc and leave this the deeper dive (like the endpoints, etc...) for this doc.  But, let's see how Lab 5 reads... it may be the equivalent of that "small number of paragraphs".